### PR TITLE
Add weights to peaq-pallet-storage

### PR DESCRIPTION
**Pull request by sayyed-kamran**

### CODE REVIEW

File: runtime/peaq-dev/src/lib.rs [hunk 1]
```diff
@@ -786,6 +787,14 @@ impl peaq_pallet_rbac::Config for Runtime {
     type WeightInfo = peaq_pallet_rbac::weights::SubstrateWeight<Runtime>;
 }
 
+
+// Config the storage in pallets/storage
+impl peaq_pallet_storage::Config for Runtime {
+	type Event = Event;
+	//type WeightInfo = peaq_pallet_storage::weights::SubstrateWeight<Runtime>;
```
Comment (sfffaaa) [inline runtime/peaq-dev/src/lib.rs]: Should we uncomment it?

File: runtime/peaq-dev/src/lib.rs [hunk 2]
```diff
@@ -786,6 +787,14 @@ impl peaq_pallet_rbac::Config for Runtime {
     type WeightInfo = peaq_pallet_rbac::weights::SubstrateWeight<Runtime>;
 }
 
+
+// Config the storage in pallets/storage
+impl peaq_pallet_storage::Config for Runtime {
```
Comment (sfffaaa) [inline runtime/peaq-dev/src/lib.rs:792]: Please also add peaq_pallet_storage to the Agung runtime module.

File: runtime/peaq-dev/src/lib.rs (hunk 1 above)
Comment (sayyed-kamran) [reply runtime/peaq-dev/src/lib.rs]: Corrected. Also reference of peaq-pallet-storage updated.

File: runtime/peaq-dev/src/lib.rs (hunk 2 above)
Comment (sayyed-kamran) [reply runtime/peaq-dev/src/lib.rs:792]: As discussed, I am going to merge this branch and will create a separate MR for adding peaq_pallet_storage to the Agung runtime module.

**Pull request merged**